css/styles.css

```diff
@@ -113,17 +112,39 @@ input:focus {
   border-radius: 25% 15%;
 }
 
+.cards {
```

Comment:
Grid isn't looking so good on mobile...
![Screen Shot 2020-02-25 at 7 59 43 PM](https://user-images.githubusercontent.com/53223556/75304658-763ed280-5809-11ea-8b37-c64e19f3119a.png)
And things aren't quite right on larger screens either...
![Screen Shot 2020-02-25 at 8 01 30 PM](https://user-images.githubusercontent.com/53223556/75304761-b1410600-5809-11ea-8f62-b10acb097044.png)
I think that the margins and padding are messing up centering or something, but I haven't really had time to learn grid very well yet, so I don't really know. Maybe it would be better to just use flexbox?

Reply:
Updated with some changes that should fix these. However, I don't see this as too big of a concern if a modal feature will be implemented to display rgb and hex values. With the modal the card will only be the box with a filled in color and will fit perfectly into whichever layout scheme we go with.

Reply:
True, these changes won't make much difference in the long run, but for now this looks much better. Thanks!